src/main/java/com/exasol/adapter/dialects/postgresql/PostgreSQLIdentifierMapping.java

```diff
@@ -33,4 +53,45 @@ public static PostgreSQLIdentifierMapping parse(final String mapping) {
                     .message("Unable to parse PostgreSQL identifier mapping from a null value.").toString());
         }
     }
+
+    /**
+     * Read identifier mapping from adapter properties.
+     *
+     * @param properties adapter properties to read identifier mapping from
+     * @return identifier mapping from properties or default value
+     */
+    public static CaseFolding from(final AdapterProperties properties) {
+        return properties.containsKey(PROPERTY) //
+                ? CaseFolding.valueOf(properties.get(PROPERTY))
+                : CaseFolding.DEFAULT;
+    }
+
+    /**
+     * @return validator for adapter properties controlling identifier mapping
+     */
+    public static PropertyValidator validator() {
+        return new Validator();
+    }
+
+    static class Validator implements PropertyValidator {
```

Comment:
```suggestion
    private static class Validator implements PropertyValidator {
```
❓ 
In tests you can use the `validator()` method to create a new instance